tests/test_modeling_utils.py

```diff
@@ -1184,3 +1186,145 @@ def test_push_to_hub_dynamic_model(self):
         config = AutoConfig.from_pretrained(f"{USER}/test-dynamic-model", trust_remote_code=True)
         new_model = AutoModel.from_config(config, trust_remote_code=True)
         self.assertEqual(new_model.__class__.__name__, "CustomModel")
+
+
+@require_torch
+class AttentionMaskTester(unittest.TestCase):
```

Comment:
would also advocate for self.assert that we use more now 😉 but it's a nit

Reply:
Hmm, I really think `assert ...` is much better to use in tests because the error message is much cleaner. E.g. if you do:

```py
self.assertTrue(expected_ids == predicted_ids)
```

assuming each is a list, you're error message will just be "the assertion is not True"

where as doing:

```py
assert expected_ids == predicted_ids
```

gives you a much better error message. Ok to change for consistency, but I really think that just doing `assert ...` is better.

Reply:
Yeah but self.assertListEqual() should give you more info no? 
I don't mine but just want consistency!